### PR TITLE
arash/footer

### PR DESCRIPTION
**Pull request by arash-fs**
# Description

Summary of the change and/or issue that is fixed.

Changes:

-   fixes issue with item
-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Refactor code
-   [ ] Update translation text
-   [ ] Breaking change
-   [ ] Dependency update
-   [ ] This change requires a documentation update
-   [ ] Release

### CODE REVIEW

File: src/components/layout/footer.js [hunk 1]
```diff
@@ -1,429 +1,267 @@
...
 `
-const LegalRow = styled.div`
-    display: grid;
-    grid-template-columns: (1, 1fr);
-    grid-column-gap: 1rem;
-    width: 100%;
-    grid-template-areas:
-        'li'
-        'lt';
+const SocialMeida = styled.section`
```
Comment (oskar-binary) [inline src/components/layout/footer.js]:
```suggestion
const SocialMedia = styled.section`
```

File: src/components/layout/footer.js [hunk 2]
```diff
@@ -1,429 +1,267 @@
...
+    <StyledFooter>
+        <Container>
+            <StyledGrid columns='repeat(12, 1fr)' columngap='2.4rem' rowgap='3.9rem'>
+                <InfoSection>
+                    <Logo width='14.5rem' />
+                    <Text>{localize('Deriv is a new trading platform created by the Binary Group, a multi-award winning pioneer in online trading.')}</Text>
+                    <SocialMeida>
+                        <Text>{localize('CONNECT WITH US')}</Text>
+                        <div>
+                            <ExternalLink href='https://www.facebook.com/binarydotcom' target="_blank" rel="noopener noreferrer">
```
Comment (oskar-binary) [inline src/components/layout/footer.js]:
```suggestion
                            <ExternalLink href='https://www.facebook.com/derivdotcom/' target="_blank" rel="noopener noreferrer">
```

File: src/components/layout/footer.js [hunk 3]
```diff
@@ -1,429 +1,267 @@
...
+import Facebook from 'images/svg/footer-facebook.svg'
 import Warning from 'images/svg/warning.svg'
+import Copyright from 'images/svg/copyright.svg'
 
-const FooterContainer = styled(Container)`
-    @media ${device.tabletL} {
-        width: 100%;
-        margin: 0;
-    }
+const StyledFooter = styled.section`
```
Comment (oskar-binary) [inline src/components/layout/footer.js]:
```suggestion
const StyledFooter = styled.footer`
```

File: src/components/layout/footer.js (hunk 1 above)
Comment (arash-fs) [reply src/components/layout/footer.js]: done

File: src/components/layout/footer.js (hunk 2 above)
Comment (arash-fs) [reply src/components/layout/footer.js]: done

File: src/components/layout/footer.js (hunk 3 above)
Comment (arash-fs) [reply src/components/layout/footer.js]: done

**Pull request merged**